### ISSUE TITLE
config backup fails execute

### ISSUE THREAD

**Issue by harche** [kimchi-project/ginger#337]
Hi,

While taking config backup the task exits with error, 
```
rc: 1 error: tar: /var/lib/ginger/ginger_backups/2887322e-b544-4aba-acb1-5877c0df5b01.tar.gz: file changed as we read it
```
The simple fix could be to always exclude the directory /var/lib/ginger/ginger_backups/ from all backups. This directory is used to store backups and if while taking custom backups it falls in the path of the directories that are being backed up you will get that error above.

**Comment by danielhb** [comment]
@harche you sent this patch to fix this issue two weeks ago:

[Ginger][PATCH] Issue #337 : config backup fails to execute


Are you planning to sent the v2?

**Comment by harche** [comment]
@danielhb Yes, I am planning to send  V2. Just getting held up in other stuff. I will send it as soon as I get time to work on it. Most probably by next week.

**Comment by danielhb** [comment]
Just checking if you was needing/wanting some helping with this. Thanks for the info
